packages/db/src/lib/granule.ts

```diff
@@ -354,3 +356,26 @@ export const getGranulesByGranuleId = async (
     .where({ granule_id: granuleId });
   return records;
 };
+
+/**
+ * Get granules from table where granule_id matches provided granuleId
+ *
+ * @param {Knex} knex - DB client or transaction
+ * @param {string} granuleId - Granule ID
```

Comment:
This param is incorrect, and doesn't match 18.5.3

Nit: Also..... we shouldn't type the docstrings in .ts for already annotated params, for both PRs.